pom.xml

```diff
@@ -292,6 +293,7 @@
                     <artifactId>scalastyle-maven-plugin</artifactId>
                     <version>1.0.0</version>
                     <configuration>
+                        <quiet>${quiet.scalastyle.check}</quiet>
```

Comment:
With `quiet` set to true by default, `scalastyle` still runs, fails on error, saves the output to a file and only prints this:
![image](https://user-images.githubusercontent.com/20905867/65050783-4214db00-d968-11e9-96fd-b4ac63d2423e.png)
If it fails the build you can find the cause of failure in the output file or rerun it with the `-Pscalastyle` profile.
It really is very lightweight, disabling it would shave no more than a single-digit seconds of build time, so I'd rather have it run always.